### PR TITLE
Added support for user-provided services at CloudFoundry.

### PR DESCRIPTION
**Pull request by mbarrientos**

### CODE REVIEW

**Comment by mbarrientos** [comment]
@rosogon, could you have a look at this? this feature is needed for Atos case study to deploy at the moment.

File: deployer/src/test/java/eu/seaclouds/location/cloudfoundry/UserProvidedServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+public class UserProvidedServiceTest extends AbstractCloudFoundryPaasLocationLiveTest {
+    private final String APPLICATION_ARTIFACT_NAME = "brooklyn-example-hello-world-webapp.war";
+    
+    private final String APPLICATION_ARTIFACT_URL =
+            getClasspathUrlForResource(APPLICATION_ARTIFACT_NAME);
+    
+    @Test(groups = {"Live"})
```
Comment (kiuby88) [inline deployer/src/test/java/eu/seaclouds/location/cloudfoundry/UserProvidedServiceTest.java]: It could move this tag to class level

**Comment by rosogon** [comment]
Hi @mbarrientos :

Thanks for the PR.

I don't have enough background to review this (I would choose @andreaturli as reviewer). The code style looks good. Is there any simple way to test this?

Is there anything pending to implement? If so, you should indicate it.

**Comment by rosogon** [comment]
Ok +1

**Pull request merged**